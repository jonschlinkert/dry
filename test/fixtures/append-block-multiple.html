{% extends "multiple" %}
{% block append="header" %}
header: "append-block-multiple.html"
{% endblock %}
{% block append="body" %}
body: "append-block-multiple.html"
{% endblock %}
{% block append="footer" %}
footer: "append-block-multiple.html"
{% endblock %}