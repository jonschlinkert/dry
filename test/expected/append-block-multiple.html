
head: "blocks/multiple.html"

header: "blocks/multiple.html"
header: "append-block-multiple.html"

body: "blocks/multiple.html"
body: "append-block-multiple.html"

footer: "blocks/multiple.html"
footer: "append-block-multiple.html"

foot: "blocks/multiple.html"
